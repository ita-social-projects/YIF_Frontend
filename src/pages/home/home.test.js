import React from 'react';
import { render } from '@testing-library/react';
import Home from './home';

test('should render correctly', () => {
  const home = render(<Home />);

  expect(home).toMatchSnapshot();
});
