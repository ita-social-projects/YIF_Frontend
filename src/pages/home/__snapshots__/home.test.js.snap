// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="header"
      >
        Header
      </div>
      <div
        class="banner"
      >
        Banner
      </div>
      <div
        class="filter"
      >
        Filter
      </div>
      <div
        class="about"
      >
        About
      </div>
      <div
        class="banner"
      >
        <div
          class="two-col-grid label"
        >
          <div
            class="grid-item wordAnimation leftWord"
          >
            Your IT
          </div>
          <div
            class="grid-item wordAnimation rightWord"
          >
            Future
          </div>
        </div>
        <div
          class="description"
        >
          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
        </div>
        <a
          class="bannerButton"
          href="/somewhere"
          rel="call to action"
        >
          Call to action
        </a>
      </div>
      <div
        class="partners"
      >
        Partners
      </div>
      <div
        class="footer"
      >
        Footer
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="header"
    >
      Header
    </div>
    <div
      class="banner"
    >
      Banner
    </div>
    <div
      class="filter"
    >
      Filter
    </div>
    <div
      class="about"
    >
      About
    </div>
    <div
      class="banner"
    >
      <div
        class="two-col-grid label"
      >
        <div
          class="grid-item wordAnimation leftWord"
        >
          Your IT
        </div>
        <div
          class="grid-item wordAnimation rightWord"
        >
          Future
        </div>
      </div>
      <div
        class="description"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
      </div>
      <a
        class="bannerButton"
        href="/somewhere"
        rel="call to action"
      >
        Call to action
      </a>
    </div>
    <div
      class="partners"
    >
      Partners
    </div>
    <div
      class="footer"
    >
      Footer
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
